### PR TITLE
SimpleFin AGE Sensor Documentation

### PR DESCRIPTION
**Pull request by jeeftor**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/122550
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Enhanced documentation for the SimpleFIN interface with a structured table format for sensor descriptions.
	- Improved clarity by detailing two sensors: `Balance` and `Age`, explaining their functionalities more effectively.

### CODE REVIEW

File: source/_integrations/simplefin.markdown [hunk 1]
```diff
@@ -30,5 +30,7 @@ Within the SimpleFIN Interface, you can create a **Claim Token**, a one-time tok
 
 Each `account` will be set up as a device in Home Assistant, and it will contain the following sensors:
 
-- `balance` 
-
+|Sensor|Description|
+|-------|---------------|
+|Balance|Account Balance|
+|Age| This sensor shows when the data was retrieved by the SimpleFin API |
```
Comment (c0ffeeca7) [inline source/_integrations/simplefin.markdown:36]: age: does this say something like 2 days old? Or does it give a timestamp?

File: source/_integrations/simplefin.markdown [hunk 2]
```diff
@@ -30,5 +30,7 @@ Within the SimpleFIN Interface, you can create a **Claim Token**, a one-time tok
 
 Each `account` will be set up as a device in Home Assistant, and it will contain the following sensors:
 
-- `balance` 
-
+|Sensor|Description|
+|-------|---------------|
+|Balance|Account Balance|
```
Comment (c0ffeeca7) [inline source/_integrations/simplefin.markdown]:
```suggestion
|Balance|Account balance|
```

File: source/_integrations/simplefin.markdown (hunk 1 above)
Comment (jeeftor) [reply source/_integrations/simplefin.markdown:36]: Here is what it looks like

Comment (c0ffeeca7) [reply source/_integrations/simplefin.markdown:36]: hm, did you add a screenshot? I can't see it.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @jeeftor :+1:

**Pull request merged**